### PR TITLE
add search tool

### LINKED ISSUE #16933: New method to find .NET global tools
I have created a free open source website to specifically search for .NET global tools:

https://www.toolget.net

Source code: https://github.com/marcusturewicz/toolget

If deemed useful, it would be worth listing this in the section "Find a .NET Core Global Tool".

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f0c5b03c-2854-8462-57dc-b0239388096e
* Version Independent ID: 13b38200-ed54-7bfe-f7cf-099cf1b010f4
* Content: [.NET Core Global Tools - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools#feedback)
* Content Source: [docs/core/tools/global-tools.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/global-tools.md)
* Product: **dotnet-core**
* Technology: **dotnet-cli**
* GitHub Login: @KathleenDollard
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: docs/core/tools/global-tools.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ Currently, there isn't a Global Tool search feature in the .NET Core CLI. The fo
 * You may find tool recommendations in blog posts or in the [natemcmaster/dotnet-tools](https://github.com/natemcmaster/dotnet-tools) GitHub repository.
 * You can see the source code for the Global Tools created by the ASP.NET team at the [dotnet/aspnetcore](https://github.com/dotnet/aspnetcore/tree/master/src/Tools) GitHub repository.
 * You can learn about diagnostic tools at [.NET Core dotnet diagnostic Global Tools](../diagnostics/index.md#net-core-dotnet-diagnostic-global-tools).
+* Use [ToolGet](https://www.toolget.net/) to search for .NET tools.
+
+> [!NOTE]
```
Comment (tdykstra) [inline docs/core/tools/global-tools.md]: The contributor guide advises against publishing statements about future product plans.

**Review by tdykstra — COMMENTED** [review]
I'm rewriting this doc -- I'll add the toolget reference to my draft and we could close this PR to avoid a merge conflict.

File: docs/core/tools/global-tools.md (hunk 1 above)
Comment (mairaw) [reply docs/core/tools/global-tools.md]: I know it's not common but people can already use this with nightly builds so I thought we could add in this case.

**Comment by mairaw** [comment]
I know you're working on this but thought we could quickly merge this PR and should be easy to merge this change to your branch. I, on purpose, didn't touch other parts of the doc that would cause conflicts.

But it's up to you. Feel free to close this if you prefer to have this on your draft instead.

**Comment by tdykstra** [comment]
I just created #17065 and included a bullet for ToolGet in it.
We usually restrict discussion of future plans to "road map" documents  that are explicitly for planning -- is there one for the CLI that we can link to, which would include the tool search feature?

**Comment by mairaw** [comment]
This would be a question for @KathleenDollard and @wli3. I'm not sure.

**Pull request closed without merging**